client/src/components/DataFiles/DataFilesListing/DataFilesListing.test.js

```diff
@@ -1,9 +1,9 @@
-import React from "react"
-import { createMemoryHistory } from "history";
-import  DataFilesListing  from "./DataFilesListing";
-import { CheckboxCell, FileNavCell } from "./DataFilesListingCells";
-import configureStore from "redux-mock-store";
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import configureStore from 'redux-mock-store';
 import renderComponent from 'utils/testing';
+import DataFilesListing from './DataFilesListing';
+import { CheckboxCell, FileNavCell } from './DataFilesListingCells';
 import systemsFixture from '../fixtures/DataFiles.systems.fixture';
 
```

Comment:
**Question**
Did anything change inhere besides lint fixes?

Reply:
Nothing changed, I think somehow auto-format changed some lines. I'm reverting the changes.

Reply:
I believe you may keep the changes. (If nothing changed, I think it's okay to slip in some lint fixes in a very related file.) @rstijerina, thoughts?

_P.S. There is now a conflict in this file._

Reply:
Yes, improved linting of test files is good with me